source/db/ar-projects.json

```diff
@@ -3461,7 +3461,6 @@
       "S/MIME",
       "SSL/TLS",
       "Tor",
```

Comment:
The trailing comma for the last element has to be removed otherwise it's not valid json. (And same goes for the rest of the files.)